h/browser/chrome/lib/hypothesis-chrome-extension.js

```diff
@@ -169,15 +180,33 @@ function HypothesisChromeExtension(dependencies) {
       return Promise.resolve();
     }
 
-    if (state.isTabActive(tab.id)) {
-      return sidebar.injectIntoTab(tab).catch(function (err) {
+    function inject(tab) {
+      sidebar.injectIntoTab(tab).catch(function (err) {
         tabErrors.setTabError(tab.id, err);
         state.errorTab(tab.id);
       });
     }
-    else if (state.isTabInactive(tab.id)) {
+
+    if (state.isTabActive(tab.id)) {
+      return uriinfo(tab.url).then(function(info) {
+        if (info.blocked) {
+            tabErrors.setTabError(
+              tab.id, new errors.BlockedSiteError(
+                "Hypothesis doesn't work on this site yet."));
```

Comment:
![untitled screencast 4](https://cloud.githubusercontent.com/assets/22498/10512985/5d2bb634-733c-11e5-958b-54efea941f47.gif)

Note that I'm clicking the browser button, but Screencastify shows the mouse cursor in the wrong location